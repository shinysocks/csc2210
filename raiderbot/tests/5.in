+--------------------+
|  TT**# ########    |
|  #T** #*    *      |
| # T**|=*.-.*#*     |
| #     #* |T*-      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  # |  # *|
|    #     ###-# #  *|
|  ##     | |S|      |
|         --- ###### |
+--------------------+
10 8
ee 
enne
nwnn
wn
wwwwwww
nnw
w
wws 



