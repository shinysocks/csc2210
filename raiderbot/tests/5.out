Initial state: Robot at 10, 8 (0 gold)
+--------------------+
|  TT**# ########    |
|  #T** #*    *      |
| # T**|=*.-.*#*     |
| #     #* |T*-      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  # |  # *|
|    #     ###-# #  *|
|  ##     |R|S|      |
|         --- ###### |
+--------------------+
========================================
Executing ee
Robot at 12, 8 (0 gold)
+--------------------+
|  TT**# ########    |
|  #T** #*    *      |
| # T**|=*.-.*#*     |
| #     #* |T*-      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  # |  # *|
|    #     ###-# #  *|
|  ##     | |R|      |
|         --- ###### |
+--------------------+
Executing enne
Robot at 14, 6 (0 gold)
+--------------------+
|  TT**# ########    |
|  #T** #*    *      |
| # T**|=*.-.*#*     |
| #     #* |T*-      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  # R  # *|
|    #     ###-# #  *|
|  ##     | |S|      |
|         --- ###### |
+--------------------+
Executing nwnn
Robot at 13, 3 (0 gold)
+--------------------+
|  TT**# ########    |
|  #T** #*    *      |
| # T**|=*.-.*#*     |
| #     #* |T*R      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  # |  # *|
|    #     ###-# #  *|
|  ##     | |S|      |
|         --- ###### |
+--------------------+
Executing wn
Robot at 12, 2 (2 gold)
+--------------------+
|  TT**# ########    |
|  #T** #*    *      |
| # T**|=*.-.R#*     |
| #     #* |T -      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  # |  # *|
|    #     ###-# #  *|
|  ##     | |S|      |
|         --- ###### |
+--------------------+
Executing wwwwwww
Robot at 5, 2 (4 gold)
+--------------------+
|  TT**# ########    |
|  #T** #*    *      |
| # T*R|= .-. #*     |
| #     #* |T -      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  # |  # *|
|    #     ###-# #  *|
|  ##     | |S|      |
|         --- ###### |
+--------------------+
Executing nnw
Robot at 4, 0 (7 gold)
+--------------------+
|  TTR # ########    |
|  #T*  #*    *      |
| # T* |= .-. #*     |
| #     #* |T -      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  # |  # *|
|    #     ###-# #  *|
|  ##     | |S|      |
|         --- ###### |
+--------------------+
Executing w
Robot at 3, 0 (6 gold)
+--------------------+
|  TR  # ########    |
|  #T*  #*    *      |
| # T* |= .-. #*     |
| #     #* |T -      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  # |  # *|
|    #     ###-# #  *|
|  ##     | |S|      |
|         --- ###### |
+--------------------+
Executing wws
Robot at 1, 1 (5 gold)
+--------------------+
|  TT  # ########    |
| R#T*  #*    *      |
| # T* |= .-. #*     |
| #     #* |T -      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  # |  # *|
|    #     ###-# #  *|
|  ##     | |S|      |
|         --- ###### |
+--------------------+
========================================
Robot has completed its task.
